### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by PouryaMoghadam**
- New cross-env package added
- cross-env added to scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
   "private": true,
   "scripts": {
-    "dev": "NODE_ENV=development node --max_old_space_size=8000 ./provider/server/development.js",
-    "build": "NODE_ENV=production webpack --config ./provider/webpack/production.js --debug --progress --display-error-details --profile --colors",
-    "start": "NODE_ENV=production node ./provider/server/production.js",
-    "start-pm2": "NODE_ENV=production pm2 start ./provider/server/production.js --name RSSR",
+    "dev": "cross-env NODE_ENV=development node --max_old_space_size=8000 ./provider/server/development.js",
+    "build": "cross-envNODE_ENV=production webpack --config ./provider/webpack/production.js --debug --progress --display-error-details --profile --colors",
+    "start": "cross-env NODE_ENV=production node ./provider/server/production.js",
+    "start-pm2": "cross-env NODE_ENV=production pm2 start ./provider/server/production.js --name RSSR",
```
Comment (PouryaMoghadam) [inline package.json:9]: Add the cross-env word at the start of each line for throw pass the conf

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.6",
     "bootstrap": "^4.3.1",
+    "cross-env": "^6.0.3",
```
Comment (PouryaMoghadam) [inline package.json:35]: added the package at this line

**Review by PouryaMoghadam — COMMENTED** [review]
This packages added for pass throw the .env configs to server files , because in node older version like mine , didn't support it by default and need package line cross-env to throw pass the configs at the script run moment

**Comment by ebrahimiaval** [comment]
We have no Merge before the public introduction.

**Pull request closed without merging**